### PR TITLE
8184134: HeapRegion::LogOfHRGrainWords is unused

### PR DESCRIPTION
**Pull request by albertnetymk**
Trivial change of removing unused code.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8184134](https://bugs.openjdk.java.net/browse/JDK-8184134): HeapRegion::LogOfHRGrainWords is unused

### Reviewers
 * [Kim Barrett](https://openjdk.java.net/census#kbarrett) (@kimbarrett - **Reviewer**) ⚠️ Review applies to 192a7a473ff694327caef469355ede9c0028ec71
 * [Stefan Johansson](https://openjdk.java.net/census#sjohanss) (@kstefanj - **Reviewer**) ⚠️ Review applies to 192a7a473ff694327caef469355ede9c0028ec71

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/3671/head:pull/3671` \
`$ git checkout pull/3671`

Update a local copy of the PR: \
`$ git checkout pull/3671` \
`$ git pull https://git.openjdk.java.net/jdk pull/3671/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 3671`

View PR using the GUI difftool: \
`$ git pr show -t 3671`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/3671.diff">https://git.openjdk.java.net/jdk/pull/3671.diff</a>

</details>

### CODE REVIEW

**Review by kimbarrett — APPROVED** [review]
Looks good.

File: src/hotspot/share/gc/g1/heapRegion.hpp [hunk 1]
```diff
@@ -318,8 +317,8 @@ class HeapRegion : public CHeapObj<mtGC> {
 
   // It sets up the heap region size (GrainBytes / GrainWords), as
   // well as other related fields that are based on the heap region
-  // size (LogOfHRGrainBytes / LogOfHRGrainWords /
-  // CardsPerRegion). All those fields are considered constant
+  // size (LogOfHRGrainBytes / CardsPerRegion).
+  // All those fields are considered constant
```
Comment (kstefanj) [inline src/hotspot/share/gc/g1/heapRegion.hpp:321]: Please reformat the comment to use similar length of lines.

**Review by kstefanj — APPROVED** [review]
Looks good, just a minor comment.

**Comment by albertnetymk** [comment]
Thanks for the review.

/integrate

**Pull request closed without merging**